.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish package
+
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+  publish-package:
+    name: Publish package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set env
+        run: echo "LIMINAL_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set up Python
```

Comment:
add git tag

Reply:
The trigger starts by git tag.
This is my condition:
```
on:
  push:
    tags:
      - '**'
```